src/mtev_console.c

```diff
@@ -269,12 +302,14 @@ mtev_console_continue_sending(mtev_console_closure_t ncct,
   eventer_t e = ncct->e;
   if(!ncct->outbuf_len) return 0;
   if(ncct->output_cooker) ncct->output_cooker(ncct);
+  OL(ncct);
```

Comment:
Do we need to lock the access to outbuf_len two lines before this?
https://github.com/circonus-labs/libmtev/blob/5812034e0fef7d34c48deedc5cb4d4f65ebe76a8/src/mtev_console.c#L303

Reply:
It's a consistency-safe race.  Should be fine.